package.json

```diff
@@ -72,5 +72,7 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
-  }
+  },
+  "main": "index.js",
+  "author": "anita-ihuman <charlesanita403@gmail.com>"
```

Comment:
If you'd like to add your name to authors I think it would be best to use an array of all collaborators

Reply:
I'd like to avoid having an author field. They add extra maintenance burden for little benefit, since we can already see the authors in the commit list and repo stats.

Reply:
Ah right, I think this is more useful for published packages